src/Microsoft.Azure.SignalR.AspNet/ServiceOptions.cs

```diff
@@ -25,6 +25,16 @@ public class ServiceOptions : IServiceEndpointOptions
         /// </summary>
         public int ConnectionCount { get; set; } = 5;
 
+        /// <summary>
+        /// Gets applicationName, which will be used as a prefix to apply to each hub name
+        /// </summary>
+        public string ApplicationName{ get; internal set; }
```

Comment:
How about using EII? For ASP.NET one, `ApplicationName` is not exposed using: 
`string IServiceEndpointOptions.ApplicationName { get; } = "";` 
or 
```cs
internal string ApplicationName { get; set; }
string IServiceEndpointOptions.ApplicationName => ApplicationName;
```

Reply:
done.